doc/classes/@GDScript.xml

```diff
@@ -304,6 +320,10 @@
 			</argument>
 			<description>
 				Returns [b]e[/b] raised to the power of 's'. [b]e[/b] sometimes called "Euler's number" is a mathematical constant whose value is approximately 2.71828.
+				[codeblock]
+				# a is 2.71828
+				a = exp(2)
```

Comment:
I would inline the comment to indicate what the value is:
a = exp(2) # approximately 7.39